packages/ecs-morgan-format/package.json

```diff
@@ -37,7 +37,8 @@
     "node": ">=10"
   },
   "dependencies": {
-    "@elastic/ecs-helpers": "^1.1.0"
+    "@elastic/ecs-helpers": "^2.0.0",
```

Comment:
Reviewer note: As mentioned at https://github.com/elastic/ecs-logging-nodejs/pull/155#issuecomment-1769521395 the tests for node <16 are expected to fail on this.

Reply:
Could we do the package linking ourselves? at least for CI?

Reply:
I don't think it is worth the complexity, frankly. We'd have to attempt the npm install, trap that it has failed, check that the `dependencies["@elastic/ecs-helpers"]` entries in each of the logger's "package.json" files points to a version that matches the version in "packages/ecs-helpers/package.json#name", then link in that case -- reproducing what `npm --workspaces ...` does.

This is only necessary for a subset of changes when there is a required dependency between the ecs-helpers and logger packages, and only for now EOL versions of node.

A perhaps better alternative would be to install `npm@7` on those older versions in CI.  I could try that. (I'm hoping I can get this in before the weekend.)

Reply:
> A perhaps better alternative would be to install `npm@7` on those older versions in CI.

That worked. Commit d87337c92e3e96678ebe5668495d497699a7da4f